doc/nrf/samples/samples_zigbee.rst

```diff
@@ -3,6 +3,78 @@
 Zigbee samples
 ##############
 
+The nRF Connect SDK provides the following samples showcasing the :ref:`ug_zigbee` protocol.
+The samples can be built for various boards and can be configured for different usage scenarios.
+
+In addition to their basic functionalities, the samples can be expanded with variants and extensions.
+The following table lists variants and extensions available for each Zigbee sample:
+
+.. list-table::
+    :widths: auto
+    :header-rows: 1
+
+    * - Variant or extension
+      - Light bulb
+      - Light switch
+      - Network coordinator
+      - NCP
+      - Shell
+      - Template
+    * - FEM support
+      - ✔
+      - ✔
+      - ✔
+      - ✔
+      - ✔
+      -
+    * - Sleepy End Device behavior
+      -
+      - ✔
+      -
+      -
+      - :ref:`** <zigbee_ug_sed>`
+      - :ref:`** <zigbee_ug_sed>`
+    * - Multiprotocol Bluetooth LE
+      - :ref:`*** <ug_multiprotocol_support>`
+      - ✔
+      - :ref:`*** <ug_multiprotocol_support>`
+      -
+      - :ref:`*** <ug_multiprotocol_support>`
+      - :ref:`*** <ug_multiprotocol_support>`
+    * - Zigbee FOTA
+      - :ref:`** <ug_zigbee_configuring_components_ota>`
+      - ✔
+      - :ref:`** <ug_zigbee_configuring_components_ota>`
+      -
+      - :ref:`** <ug_zigbee_configuring_components_ota>`
+      - :ref:`** <ug_zigbee_configuring_components_ota>`
+    * - Endpoint logger
+      - :ref:`** <ug_zigbee_configuring_components_logger_ep>`
+      - :ref:`** <ug_zigbee_configuring_components_logger_ep>`
+      - :ref:`** <ug_zigbee_configuring_components_logger_ep>`
+      -
+      - ✔
+      - :ref:`** <ug_zigbee_configuring_components_logger_ep>`
+    * - ZCL scene helper
+      - ✔
+      - :ref:`*** <ug_zigbee_configuring_components_scene_helper>`
+      - :ref:`*** <ug_zigbee_configuring_components_scene_helper>`
+      -
+      - :ref:`*** <ug_zigbee_configuring_components_scene_helper>`
+      - :ref:`*** <ug_zigbee_configuring_components_scene_helper>`
```

Comment:
I don't think that `***` is correct here - enabling `ZCL scene library` should have the same level of difficulty as `Zigbee FOTA`.
Also, for most samples it doesn't make sense to enable it as those samples don't implement cluster with appropriate cluster roles that `ZCL Scenes` utilizes.

Reply:
@tomchy , any opinion?

Reply:
I still think that ZCL scenes are more difficult to use:
 - FOTA can be enabled on a standalone endpoint and most of the device definition does not have to be modified.
 - FOTA support is universal, for ZCL scenes you have to check if all clusters are supported by the helper library, and if not, extend it by some additional logic.
 - ZCL scenes library applies only for the devices that need the server role of the cluster, so you have to think it through before adding it to the app.

Reply:
No changes made.